lightning/src/chain/mod.rs

```diff
@@ -167,12 +167,12 @@ pub trait Confirm {
 	/// given to [`transaction_unconfirmed`].
 	///
 	/// If any of the returned transactions are confirmed in a block other than the one with the
-	/// given hash, they need to be unconfirmed and reconfirmed via [`transaction_unconfirmed`] and
-	/// [`transactions_confirmed`], respectively.
+	/// given hash at the given height, they need to be unconfirmed and reconfirmed via
+	/// [`transaction_unconfirmed`] and [`transactions_confirmed`], respectively.
 	///
 	/// [`transactions_confirmed`]: Self::transactions_confirmed
 	/// [`transaction_unconfirmed`]: Self::transaction_unconfirmed
-	fn get_relevant_txids(&self) -> Vec<(Txid, Option<BlockHash>)>;
+	fn get_relevant_txids(&self) -> Vec<(Txid, u32, Option<BlockHash>)>;
```

Comment:
Might be worth adding a `TransactionConfirmation` struct at some point and rename the method.